content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -79,72 +72,97 @@ npm install
 ```terminal
 ls -1
 README.md
-functions
+functions/
 netlify.toml
-node_modules
+node_modules/
 package.json
-prisma
-public
+prisma/
+public/
 schema.sql
 ```
 
-## 3. Set the DATABASE_URL environment variable locally
+## 3. Get the Heroku PostgreSQL connection string
+
+You need the Heroku PostgreSQL connection string to define the necessary Prisma connection variable.
+
+1. In Heroku, open your app and select the **Resources** tab.
+2. Click the **Heroku Postgres** database add-on.
+3. Select the **Settings** tab in the page of the Heroku Postgres add-on.
+4. Click **View Credentials**.
+5. From the **URI** field, copy the entire connection string.
 
-Set the `DATABASE_URL` environment variable locally so you can create the database schema and Prisma can access the database to introspect:
+**Checkpoint:** The Heroku connection string has the following format:
+
+```
+postgres://__USER__:__PASSWORD__@__HOST__/__DATABASE__
+```
+
+When you next set the `DATABASE_URL` environment variable, you need to change the beginning string from `postgres` to `postgresql`.
```

Comment:
I think using Heroku is only one option mentioned in the introduction of the tutorial, you can use _any_ database provider. Making it its own headline here, changes that a bit. I think it might be easier to add this as an optional "If you want to use Heroku, this is how you do it" step here.

Reply:
I've added the optional wording related to Heroku.